src/Tizen.MachineLearning.Inference/Tizen.MachineLearning.Inference/Commons.cs

```diff
@@ -262,6 +262,12 @@ public enum HWType
         NPUSR = 0x13000,
     }
 
+    public enum ServiceEventType
+    {
+        Unknown = 0,
```

Comment:
Recommend removing unknown type. 
see PipelineState : it does not contain UNKNOWN, I received a review to remove negative value (e.g. unknown, null) in the past.

Reply:
Removed unknown type
Thank you :)